### ISSUE TITLE
Sub-menus open scrolled down by the same amount as the top-level menu

### ISSUE THREAD

**Issue by alansegar** [gantry/gantry5#2048]
If the screen / window is small enough that the user has to scroll down to get to some of the top-level menu items then when the sub-menu opens, it is scrolled down by the same amount meaning that sometimes all the sub-menu items are hidden off the top of the screen.
Desired behaviour: The sub-menus open fully scrolled to the top rather than inheriting the same amount of scroll that the top-level menu has.

Can be seen on http://www.demo-gantry5.en-toutes-lettres.fr/en/helium/helium-particles
Affects both desktop browser with mouse (if the window is small enough to trigger the mobile menu) and mobile with touch but mobile is worst affected due to the vertical scrollbar being hidden which means the user sometimes sees a blank sub-menu with no indication that they need to (or even can) scroll up.

**Comment by mahagr** [comment]
So you mean that if you scroll down in the mobile menu and then get to the sub-menu, you cannot see anything without scrolling back up?

**Comment by alansegar** [comment]
@mahagr That's correct. The further down you have to scroll on the top-level menu to get to the item you want, the more of the sub-menu is hidden until you scroll up. In the worst case, the entire sub-menu is hidden until you scroll up and as there's no scroll bar on mobile it's not obvious to the user that they need to scroll up.

**Comment by JoomFX** [comment]
Hey @mahagr and @w00fz ,

I can confirm this issue.
Have a look at [this video](https://youtu.be/VtUPoderUEc). It shows the issue in details.

**Comment by Ingrid53** [comment]
@JoomFX 
I also can confirm this issue.
Same as shown in your video

**Comment by alansegar** [comment]
Is there any plan to fix this? I'm still able to see the issue on v5.4.23 and it really spoils the usability of menus that are affected. Users don't realise that there are other items in the sub-menus due to them being hidden and there's no indication that they need to scroll in order to see them.

**Comment by pnutster** [comment]
Also dealing with this issue, any progress on this one?
